mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/Analytics.java

```diff
@@ -75,10 +71,11 @@
     /**
      * Create a graph computer from a Mindmaps Graph. The computer operates on all instances in the graph.
      */
-    public Analytics() {
-        graph = MindmapsClient.getGraph(keySpace);
-        transaction = graph.getTransaction();
-        computer = MindmapsClient.getGraphComputer(keySpace);
+    public Analytics(String keySpace) {
+        this.keySpace = keySpace;
+        graph = MindmapsClient.getGraph(this.keySpace);
```

Comment:
Instead of using MindmapsClient, maybe these constructors should accept a MindmapsGraph